requirements/edx/coverage.in

```diff
@@ -12,5 +12,5 @@
 
 -c ../constraints.txt
 
-coverage==5.0a6                     # Code coverage testing for Python
+coverage==5.0b1                     # Code coverage testing for Python
```

Comment:
Could we use something like `coverage >= 5.0b1`, so that we get future beta updates?

Reply:
I'd have to look into whether that would get us pre-releases. It might make us wait for "5.0".